### DISCUSSION TITLE
Tablist Order

### DISCUSSION THREAD

**Discussion by Teyir** [Zrips/CMI#4695] [Feature Requests]
Hello,

Can you make a new feature for order the tablist withe the rank?

Exemple:

Set a new settings like that:

```
RankPriority:
 - Admin
 - Moderator
 - VIP
 - Default
```

Or with a permission.

Why not withe a weight.

If I have the permission : _cmi.tablist.priority.1_ I'm on the top, if I have _cmi.tablist.priority.2_ I'm second etc...

Thank you very much :)

**Comment by mrfloris** [comment]
zrips, this really is one of the top 10 requested features, I hope one day it's possible;
https://github.com/Zrips/CMI/issues/931

**Comment by Skyfighter1** [comment]
Yes pls....
